packages/salesforcedx-vscode-replay-debugger/src/commands/batchDeleteExistingOverlayActionsCommand.ts

```diff
@@ -0,0 +1,51 @@
+import { BaseCommand } from '@salesforce/salesforcedx-utils-vscode/out/src/requestService';
+import { RestHttpMethodEnum } from '@salesforce/salesforcedx-utils-vscode/out/src/requestService';
+
+// The batch requests is pretty straightforward, it's an array of single requests
+export interface BatchRequests {
+  batchRequests: BatchRequest[];
+}
+
+export interface BatchRequest {
+  method: RestHttpMethodEnum;
+  url: string;
+}
+
+// The batch response is a bit nuts. There's an array of results which contain an status code and a result however
```

Comment:
Remove snarky comments. This is an open source project and you’re representing Salesforce.

Reply:
done